src/locales/help/helpDE.json

```diff
@@ -0,0 +1,11 @@
+{
+  "help": {
+    "title": "Hilfe",
+    "about_the_app": "Über die App",
```

Comment:
So wie du es hier definierst, so musst du auch in der translation methode `t(..)` bzw `$t(..)` darauf zugreifen. Möchtest du den String für about die app haben machst du `$t(help.about_the_app)`. So wie du darauf zugreifst oben funktioniert es nicht.